scylladb/sphinx-scylladb-theme
# fix: anchor links top scroll

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/889

Related issue https://github.com/scylladb/scylladb/issues/15420

Handles top scroll padding at the html level (full page) instead of per component type (e.g. sections, glossary).

## How to test

1. Build docs
2. Click on any anchor link:

    ![image](https://github.com/scylladb/sphinx-scylladb-theme/assets/9107969/0289f33d-b46f-4b42-9671-31e6fae39663)

3. It should scroll after the header + 10px.
